### PR TITLE
small bugfixes

### PR DESCRIPTION
**Pull request by mrshirts**

### CODE REVIEW

File: intermol/gromacs/gromacs_parser.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def default_gromacs_include_dir():
     if 'GMXBIN' in os.environ:
         return os.path.abspath(os.path.join(
             os.environ['GMXBIN'], '..', 'share', 'gromacs', 'top'))
-    return '/usr/local/gromacs/share/gromacs/top'
+    return '/usr/local/share/gromacs/top'
```
Comment (ctk3b) [inline intermol/gromacs/gromacs_parser.py]: Are you sure about this? On my laptop, I have gromacs in the default install location and it's `/usr/local/gromacs...`

Comment (mrshirts) [inline intermol/gromacs/gromacs_parser.py]: This might be a change for 5.0?  It appears to be where it was installed then.

Comment (mrshirts) [inline intermol/gromacs/gromacs_parser.py]: Maybe it needs to be detected more automatically?

Comment (ctk3b) [inline intermol/gromacs/gromacs_parser.py]: I'll check later tonight but if that was indeed changed moving to 5.x then we should add a check for if the directory exists and prefer 5.x over 4.x

**Pull request merged**